cmds/free/free.go

```diff
@@ -0,0 +1,241 @@
+// Copyright 2012-2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// free reports usage information for physical memory and swap space.
+//
+// Synopsis:
+//     free [-k] [-m] [-g] [-t] [-h] [-json]
+//
+// Description:
+//     Read memory information from /proc/meminfo and display a summary for
+//     physical memory and swap space. The unit options use powers of 1024.
+//
+// Options:
+//     -k: display the values in kibibytes
+//     -m: display the values in mebibytes
+//     -g: display the values in gibibytes
+//     -t: display the values in tebibytes
+//     -h: display the values in human-readable form
+//     -json: use JSON output
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+)
+
+var humanOutput = flag.Bool("h", false, "Human output: show automatically the shortest three-digits unit")
```

Comment:
super nit, but mind grouping these? Just like the const below.

```go
var (
   ....
)
```

Reply:
too late, it was merged already :-( I can make another PR